hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/OzoneConfigKeys.java

```diff
@@ -57,6 +57,12 @@
   public static final boolean DFS_CONTAINER_IPC_RANDOM_PORT_DEFAULT =
       false;
 
+  public static final String DFS_CONTAINER_RATIS_DATASTREAM_IPC_RANDOM_PORT =
+      "dfs.container.ratis.datastream.ipc.random.port";
```

Comment:
Let's remove "ipc", i.e. dfs.container.ratis.datastream.random.port".  It is not really an ipc.